machine.go

```diff
@@ -454,17 +513,23 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 		w.WriteDirectory("/lib", 0755)
 	}
 
-	prefix := ""
 	if mergedUsrSystem() {
-		prefix = "/usr"
+		w.CopyFile("/usr/bin/busybox")
```

Comment:
On Fedora busybox is under `/usr/sbin/busybox`. Maybe search `PATH` for it instead of checking only `/bin` and `/usr/bin`?

Reply:
Good point, thanks :)

Reply:
opened that as a separate issue ; it doesn't block this MR: https://github.com/go-debos/fakemachine/issues/48